src/cmd/inspector/main.go

```diff
@@ -56,6 +64,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	if inspectionPolicy.Spec.Inspection.Assessment.Governor.Enabled {
+		err, cspApiToken := getCSPTokenFromSecret(ctx, cspSecretNamespace, inspectionPolicy.Spec.Inspection.Assessment.Governor.CspSecretName)
```

Comment:
Do not need to pass **cspSecretNamespace** as args if this is global const.

Reply:
I want to move this function in csp util later on, hence created the function in a way that its self dependent.